config/bidderinfo.go

```diff
@@ -356,6 +358,44 @@ func validateMaintainer(info *MaintainerInfo, bidderName string) error {
 	return nil
 }
 
+func validateAliasCapabilities(bidder BidderInfo, infos BidderInfos, bidderName string) error {
+	if len(bidder.AliasOf) > 0 {
+		if parentBidder, ok := infos[bidder.AliasOf]; ok {
+			if parentBidder.Capabilities == nil && bidder.Capabilities != nil {
+				return fmt.Errorf("alias capabilities should be a subset of parent bidder capabilities")
+			}
+			if (parentBidder.Capabilities.Site == nil && bidder.Capabilities.Site != nil) || (parentBidder.Capabilities.Site != nil && bidder.Capabilities.Site == nil) {
+				return fmt.Errorf("alias capabilities should be a subset of parent bidder capabilities")
+			} else if bidder.Capabilities.Site != nil {
+				return validateAliasPlatformInfo(parentBidder.Capabilities.Site, bidder.Capabilities.Site)
```

Comment:
can pass  `*parentBidder.Capabilities.Site` `*bidder.Capabilities.Site` to `validateAliasPlatformInfo`

Reply:
Updated in [ba622f5](https://github.com/prebid/prebid-server/pull/2996/commits/ba622f5d216bfead0ca7beea29b32ba27eb552d6)